src/streaming/controllers/ProtectionController.js

```diff
@@ -439,20 +496,26 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this[MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_STATUSES_CHANGED] = onKeyStatusesChanged.bind(this);
 
             keySystems = this.protectionExt.getKeySystems();
-            this.protectionModel = this.system.getObject("protectionModel");
-            this.protectionModel.init();
+
+            // For browsers that don't support any version of EME, the app crashes without this check
+            if (this.system.hasMapping("protectionModel")) {
+                this.protectionModel = this.system.getObject("protectionModel");
+                this.protectionModel.init();
+            }
 
```

Comment:
@rashmimittal Agreed.  However, applications should not be passing in ProtectionControllers unless the user agent supports EME.  A different check needs to be implemented in `MediaPlayer` to throw an error if an application tries to do this.

Unfortunately, GitHub won't let me add comments to files that you didn't change, so I'll try to add the information about where to change Stream.js in this thread.